test/queue-test.js

```diff
@@ -370,6 +378,31 @@ describe('Queue', (it) => {
 
         t.context.handleErrors(t);
       });
+
+      it('should not quit the command client by default if given in settings', async (t) => {
+        const client = await redis.createClient();
+
+        sinon.spy(client, 'quit');
+
+        let queue = t.context.makeQueue({
+          redis: client
+        });
+
+        await queue.close();
+
+        t.true(client.ready);
+        t.false(client.quit.called);
+
+        queue = t.context.makeQueue({
+          redis: client,
+          quitCommandClient: true
+        });
+
+        await queue.close();
```

Comment:
do we test (or care about) the case where we pass `quitCommandClient: false` but don't also provide an existing redis client?